uring/src/main/scala/fs2/io/uring/net/UringDatagramSocket.scala

```diff
@@ -117,9 +114,24 @@ private[net] object UringDatagramSocket {
   def apply[F[_]](ring: Uring[F], fd: Int)(implicit
       F: Async[F]
   ): Resource[F, UringDatagramSocket[F]] =
-    ResizableBuffer(65535).evalMap { buf =>
-      (Mutex[F], Mutex[F]).mapN(new UringDatagramSocket(ring, fd, buf, 65535, _, _))
-    }
+    ResizableBuffer(65535)
```

Comment:
Why did we go back to the resizable buffer? I think we can just allocate this with malloc along with the `iov` and `msghdr`.

Reply:
Yes, I still have to work on the buffer